Week-5/Assignment4.test.js

```diff
@@ -0,0 +1,15 @@
+const mathOperations = require('./Assignment4')
+
+describe("Math tests",() => {
+    test('Addition of 4 and 6 should return 10', () => { 
+        expect(mathOperations.sum(4,6)).toBe(10);
+    });
```

Comment:
[For your learning] Add some negative test cases as well.. for ex : if you pass string to add method as in sum('1','2') result will be 12 right.. so add a functionality in the code such that data type is only numbers.. if the parameters are string return "number are not integers".. add this test case as well..